app/plugins/prometheus.md

```diff
@@ -0,0 +1,164 @@
+---
+id: page-plugin
+title: Plugins - Prometheus
+header_title: Prometheus
+header_icon: /assets/images/icons/plugins/prometheus.png
+breadcrumbs:
+  Plugins: /plugins
+nav:
+  - label: Usage
+    items:
+      - label: Log Format
+      - label: Kong Process Errors
+description: |
+  This plugin exposes metrics in [Prometheus Exposition format](https://github.com/prometheus/docs/blob/master/content/docs/instrumenting/exposition_formats.md). It allows you to integrate with your existing monitoring and alerting system.
+
+params:
+  name: prometheus
+
+---
+
+
+#### Enable the plugin
+
+```bash
+$ curl http://localhost:8001/plugins name=prometheus
+```
+
+#### Reading metrics into Prometheus
+
+Metrics are availble on the admin API at `/metrics` endpoint:
```

Comment:
Kong entities like "Admin API" are capitalized.

Reply:
You may want to note that this response is only an example - different configurations of Kong will result in different response content. Or if that is obvious, no change needed.